src/Slider.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import {shallow, mount} from 'enzyme'
+
+
+describe('Slider', () => {
+  it('should render without throwing', () => {
+    shallow(<Slider />)
```

Comment:
I'd wrap it in an expect func and use the assertion api, like this `expect(shallow(<Slider />).not.to.throw();` 

I think a render error would still cause the suite to fail, but the failure output might not be as clear about what the problem was.

Reply:
also, you forgot to import `Slider`, which is what's causing the CI to fail